### ISSUE TITLE
react-query failing with Cannot find module '.../vike-react/examples/react-query/node_modules/vike-react-query/dist/renderer/StreamedHydration

### ISSUE THREAD

**Issue by stepan-romankov** [vikejs/vike-react#49]
To reproduce:
- Clone repository
- change dependency versions in examples/react-query/package.json to 
    "vike-react": "0.3.7",
    "vike-react-query": "0.0.1",
- run `npm run dev`
- visit `http://localhost:3000/`
- check logs in the output

```Error when evaluating SSR module virtual:vike:pageConfigValuesAll:server:/pages/_error: failed to import "vike-react-query/renderer/VikeReactQueryWrapper"
|- Error: Cannot find module 'vike-react/examples/react-query/node_modules/vike-react-query/dist/renderer/StreamedHydration' imported from vike-react/examples/react-query/node_modules/vike-react-query/dist/renderer/VikeReactQueryWrapper.js
    at finalizeResolution (node:internal/modules/esm/resolve:264:11)
    at moduleResolve (node:internal/modules/esm/resolve:917:10)
    at defaultResolve (node:internal/modules/esm/resolve:1130:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36)```

**Comment by stepan-romankov** [comment]
Could that be that StreamedHydration.js is missing in the package.json "exports" ?

**Comment by brillout** [comment]
Maybe, would you up for a PR? The code is quite small, it should be fairly easy to fix.

**Comment by stepan-romankov** [comment]
That was only my guess... I'm actually a Golang backend developer and just making PoC for my side project... 
btw I see another error before this one
```Error when evaluating SSR module /pages/index/Movies.tsx: failed to import "vike-react-query"
|- Error: Cannot find module 'vike-react/examples/react-query/node_modules/vike-react-query/dist/src/withFallback' imported from vike-react/examples/react-query/node_modules/vike-react-query/dist/src/index.js```

**Comment by brillout** [comment]
I'll have a look at it in moment.

**Comment by brillout** [comment]
I could reproduce, let me fix it.

**Comment by brillout** [comment]
Your errors are fixed. But there is another issue, let me fix that one to.

**Comment by brillout** [comment]
Fixes released in `0.0.2`. I tried and and it now works.

@nitedani I had to remove the `import.meta.env.DEV` checks: https://github.com/vikejs/vike-react/commit/5b7ceb769c43a60f9c10978f989099972b6ac6cc.
